libobs/media-io/video-frame.c

```diff
@@ -212,6 +244,32 @@ void video_frame_init(struct video_frame *frame, enum video_format format,
 		frame->linesize[3] = width;
 		break;
 	}
+
+	case VIDEO_FORMAT_YA2L: {
+		const uint32_t plane_size = width * height * 2;
```

Comment:
Minor change to avoid redundant multiply. I would also prefer linesize to plane_linesize.
```
const uint32_t linesize = width * 2;
const uint32_t plane_size = linesize * height;
```